examples/placeholders.ts

```diff
@@ -0,0 +1,30 @@
+import * as Z from "../mod.ts";
+
+const first_ = Symbol();
+const first = Z._<string>()(first_);
+
+const second_ = Symbol();
+const second = Z._<number>()(second_);
+
+const third_ = Symbol();
+const third = Z._<boolean>()(third_);
+
+const run = Z.runtime({
+  // TODO: disable repeats
+  apply: [
+    first("HELLO"),
+  ],
+});
```

Comment:
This is super useful.

There may be a case to compose runtimes
For example,
```
const run1 = Z.runtime({
  apply: [
    first("HELLO"),
  ],
});
const run2 = Z.runtime({
  apply: [
    second(100),
  ],
});
const result = await compose(run1, run2)(root, (_) =>
  _(
    third(true),
  ));
```

Reply:
I think we want to discourage lib developers from exposing their own runtimes. Ideally, every app will instantiate their own with the necessary props (including any global placeholder application).